Factory/Config/Filter/AutoRotateFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Factory\Config\Filter;
+
+use Liip\ImagineBundle\Config\Filter\Type\AutoRotate;
+use Liip\ImagineBundle\Config\FilterInterface;
+use Liip\ImagineBundle\Factory\Config\FilterFactoryInterface;
+
+final class AutoRotateFactory implements FilterFactoryInterface
+{
+    const NAME = 'auto_rotate';
```

Comment:
as we are aiming for php 7.1, we should also qualify the visibility of these constants. i guess they should be public.

and imho, they should be in the Filter class, not the factory. and remove the $name constructor argument in favor of using the constant.